src/components/CommentList/CommentList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../../types/Comment';
+
+import './CommentList.scss';
+
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => {
+  return (
```

Comment:
mentor suggestion is to write like this:
```suggestion

export const CommentList: React.FC<Props> = (props) => {
  const { comments } = props;

```